src/calculator.js

```diff
@@ -6,7 +6,7 @@ exports._check = () => {
   
```

Comment:
Trailing spaces not allowed.

###### See http://eslint.org/docs/rules/no-trailing-spaces for details about this rule